recipes-bsp/grub/grub_%.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI_prepend := "https://github.com/3mdeb/grub2/commit/3425b85e83dbea3dffa0f88d4c2051c771ebc033"
```

Comment:
@Asiderr This wont do anything useful and you can remove  this file. We will work on GRUB on this branch: https://github.com/3mdeb/meta-pcengines/pull/2
Let's focus on Xen verification